packages/kit/src/core/utils.js

```diff
@@ -106,3 +106,20 @@ function find_svelte_packages(cwd) {
 export function get_no_external(cwd, user_specified_deps = []) {
 	return [...user_specified_deps, ...find_svelte_packages(cwd)];
 }
+
+/**
+ * Decides how the body should be parsed based on its mime type.
+ *
+ * This is intended to be used with both requests and responses, to have a consistent body parsing across adapters.
+ *
+ * @param {string} content_type The `content-type` header of a request/response.
+ * @returns {"string"|"buffer"}
+ */
+export function get_body_type(content_type) {
```

Comment:
We use underscores in names for internal methods and camelCase for public APIs. I think this should be camelCase just as `getRawBody` is since it's exposed to all adapter authors.

How about making it `isBodyBinary` and just returning a `boolean`? `buffer` vs `string` sounds slightly odd to me

Reply:
Ok, I'll make camelCase.
Yep it feels weird, I just made it an enum because it may be expanded on the future

Reply:
yeah, that's what I figured. I'm not too worried about expanding. Worst case we can just make a new method and deprecate the old one